### ISSUE TITLE
Checkbox focus outline is not per Fiori specs

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-styles#2858]
The offset for the focus outline of checkbox is 2px and the specs say it should be 1px. Need to investigate further why it was created with 2px offset and if it's a bug we need to make it as per specs.
